libs/s25main/buildings/nobMilitary.cpp

```diff
@@ -136,6 +153,8 @@ void nobMilitary::Serialize(SerializedGameData& sgd) const
     sgd.PushObjectContainer(ordered_coins, true);
     sgd.PushObjectContainer(troops, true);
     sgd.PushObjectContainer(far_away_capturers, true);
+    for(unsigned n : desired_troops)
+        sgd.PushUnsignedInt(n);
```

Comment:
```suggestion
    sgd.PushContainer(desired_troops);
```

Reply:
I can't find this function, I used the one from `helpers` instead.